src/browser/api/external_window.ts

```diff
@@ -532,10 +534,13 @@ function subscribeToWinEventHooks(externalWindow: Shapes.ExternalWindow): void {
 
 // Window grouping stub (makes external windows work with our original disabled frame group tracker)
 // Also some of the _options' values are needed in OpenFin Layouts
-function setAdditionalProperties(externalWindow: Shapes.ExternalWindow, properIdentity: Shapes.NativeWindowIdentity): Shapes.GroupWindow {
+function setAdditionalProperties(
+  externalWindow: Shapes.ExternalWindow,
+  requestedIdentity: Shapes.NativeWindowIdentity
+): Shapes.GroupWindow {
   const { nativeId } = externalWindow;
-  const uuid = properIdentity.uuid || nativeId;
-  const name = properIdentity.name || nativeId;
+  const uuid = externalWindow.uuid || requestedIdentity.uuid || nativeId;
```

Comment:
Thanks for catching this. Actually not a problem because we ensure that we have a usable uuid before we call this function. This does mean that these `||` are unnecessary. Removed them accordingly.